Shared/Samples/Set max extent/SetMaxExtentView.swift

```diff
@@ -41,11 +41,30 @@ struct SetMaxExtentView: View {
     /// A Boolean value indicating whether the max extent is set.
     @State private var maxExtentIsSet = true
     
+    /// A Boolean value indicating whether the settings view is showing.
+    @State private var showSettings = false
+    
     var body: some View {
         MapView(map: map, graphicsOverlays: [graphicsOverlay])
             .toolbar {
                 ToolbarItem(placement: .bottomBar) {
-                    Toggle(maxExtentIsSet ? "Max Extent Enabled" : "Max Extent Disabled", isOn: $maxExtentIsSet)
+                    Button("Settings") {
+                        showSettings.toggle()
+                    }
+                    .popover(isPresented: $showSettings) {
+                        settings
+                            .frame(idealWidth: 350, idealHeight: 300)
```

Comment:
I see the `idealWidth` vary among places. Can you keep them consistent for the settings form with similar content? For example, if the settings pages only have a toggle, they can have the same width and height. If the pages have multiple sections including segmented toggle, steppers, etc., they can have the same width.

This is not a big deal, but I prefer consistency for the settings pages among samples.

Also, no need to update the screenshots in the description after a review. Let's only update the final screenshots once all UI feedback have been addressed.